Dockerfile

```diff
@@ -28,10 +28,21 @@ ARG YARN_VERSION=1.22.22
 # enable corepack and precache yarn and pnpm
 RUN set -ex; \
   corepack install --global pnpm@${PNPM_VERSION} yarn@${YARN_VERSION}; \
-  pnpm --version; \
-  yarn --version; \
+  COREPACK_ENABLE_DOWNLOAD_PROMPT=0 pnpm --version; \
+  COREPACK_ENABLE_DOWNLOAD_PROMPT=0 yarn --version; \
   true
 
+USER 1000
```

Comment:
Maybe we should add some comment why we change user too 1000 ?

Reply:
it's a workaround and will be removed on next major by a better fix